stable/prometheus-node-exporter/templates/daemonset.yaml

```diff
@@ -2,27 +2,22 @@ apiVersion: extensions/v1beta1
 kind: DaemonSet
 metadata:
   name: {{ template "prometheus-node-exporter.fullname" . }}
-  labels:
-    app: {{ template "prometheus-node-exporter.name" . }}
-    chart: {{ template "prometheus-node-exporter.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+  labels: {{ include "prometheus-node-exporter.labels" . | indent 4 }}
 spec:
   selector:
     matchLabels:
-      app: {{ template "prometheus-node-exporter.name" . }}
-      release: {{ .Release.Name }}
+      {{ include "prometheus-node-exporter.labels" . | indent 6 }}
```

Comment:
Selectors should only have `app` and `release` labels.

Reply:
I really like your pull requests review because I can always learn the best practices from helm. Could you point me to a document where this is explained?

Reply:
I don't have a link at hand, but selectors should be stable which is not the case for, e. g., the chart label when you update the chart. This might get you in trouble.